plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -393,15 +388,12 @@ object ScalafmtPlugin extends AutoPlugin {
 
   private def scalafmtTask(sources: Seq[File], session: FormatSession) =
     Def.task {
-      session.formatTrackedSources(streams.value.cacheStoreFactory, sources)
```

Comment:
would changing this to `(scalafmt / streams).value.cacheStoreFactory` be an equivalent single-line fix? if not, can you explain how this change actually works?

Reply:
i tested this, doesn't work. mystery...

Reply:
It wouldn't since we would still be missing the configuration axis (not propagated from the taskDyn, see https://github.com/scalameta/sbt-scalafmt/pull/192#issuecomment-1000564191) when resolving the `.value` lookup. 

However, an equivalent fix would be to pass the config as an argument of the method, and use `(config / scalafmt / streams).value.cacheStoreFactory`, but I felt like capturing the cache directory in the session was simpler.